### ISSUE TITLE
Red Solstice 2: Survivors (768520)

### ISSUE THREAD

**Issue by coreybruce** [ValveSoftware/Proton#5533]
# Compatibility Report
- Name of the game with compatibility issues: Red Solstice 2: Survivors
- Steam AppID of the game: 768520

## System Information
System info:
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/coreybruce/dcd2bec911594c9212d351ac2e7a3f98
- Proton version: Proton-experimental

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.

## Symptoms 

Game will crash while loading campaign using proton experimental 

note: the game does work with proton 6.3-8 and will be fine so that is a work around but I wanted to report this to fix the bug on  proton experimental 

## Reproduction
Run the game using proton experimental and try and do the campaign

Log as gist: https://gist.github.com/coreybruce/fe6718fbf1fd24a657edd79da7113fc3

**Comment by alasky17** [comment]
@coreybruce Could you see if you get the crash in experimental if you are starting a brand new campaign that forces you to complete the tutorial first?  I'm not seeing the crash, but I don't have a real saved game yet.

**Comment by coreybruce** [comment]
> @coreybruce Could you see if you get the crash in experimental if you are starting a brand new campaign that forces you to complete the tutorial first? I'm not seeing the crash, but I don't have a real saved game yet.

Yeah it does, that is what my report is about :)  if I untick the tutorial and do the first mission it will crash on the loading screen. if I switch to proton 6.3-8 it will work fine and not crash

**Comment by alasky17** [comment]
@coreybruce Thank you - I was testing on AMD, so possibly its a driver difference.  Is this still happening on 7.0-1 and/or experimental-7.0?

**Comment by coreybruce** [comment]
> @coreybruce Thank you - I was testing on AMD, so possibly its a driver difference. Is this still happening on 7.0-1 and/or experimental-7.0?

Thanks for the reply! no, I believe they did eventually fix this in experimental as  I was able to get in and play without crashing after a update some time ago and it works without crashing proton 7.0.1.

Note tho idk if it's the game or proton but it will sit around 50fps might drop to 40fps on high settings and vsync with my hardware
